listenbrainz/webserver/static/js/src/playlists/Playlist.tsx

```diff
@@ -29,8 +29,8 @@ import {
 } from "../AlertNotificationsHOC";
 import APIServiceClass from "../APIService";
 import GlobalAppContext, { GlobalAppContextT } from "../GlobalAppContext";
-import SpotifyAPIService from "../SpotifyAPIService";
-import BrainzPlayer from "../BrainzPlayer";
+import SpotifyAPIService from "../brainzplayer/SpotifyAPIService";
```

Comment:
As mentioned, this is going to need reverting

Reply:
Done!